spec/versionedsignedbeaconblock.go

```diff
@@ -66,6 +66,29 @@ func (v *VersionedSignedBeaconBlock) Slot() (phase0.Slot, error) {
 	}
 }
 
+// BlockHash returns the block hash of the beacon block.
+func (v *VersionedSignedBeaconBlock) BlockHash() (phase0.Hash32, error) {
```

Comment:
This should be `ExecutionBlockHash` as per the versioned signed blinded beacon block commit previously.